### ISSUE TITLE
Runtime environment

### ISSUE THREAD

**Issue by tuzhirun** [robotics/open_abb#24]
First of all, thank you very much for your work.This communication file abb. Py can also run on Windows, right?Is all you need to run the system is server.mod and logger.mod, abb.py? Do you only need these three files?

**Comment by milistu** [comment]
Yes, it can run on Windows. You only need the files that you specified. It will also work with server.mod and abb.py, if you don't need outputs from RobotStudio to appear in Python.
